plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -74,8 +73,21 @@ function handleElement($node) {
 		$issue->setCurrent($node->getAttribute('current'));
 		$issue->setAccessStatus($node->getAttribute('access_status'));
 
-		$issueDao->insertObject($issue);
-		$deployment->setIssue($issue);
+		$foundIssue = null;
+		foreach ($node->getElementsByTagName('issue_identification') as $n) {
+			$searchIssue = $issueDao->newDataObject();
+			$this->parseIssueIdentification($n, $searchIssue, false);
```

Comment:
How about letting this function return either an issue, if such an issue is found, or null? So that we would not create newDataObject above, but first below, if returned issue is null?